token/core/fabtoken/validator.go

```diff
@@ -167,6 +170,12 @@ func (v *Validator) verifyIssue(issue driver.IssueAction) error {
 }
 
 func (v *Validator) VerifyTransfer(inputTokens []*token2.Token, tr driver.TransferAction) error {
+	if inputTokens[0] == nil {
```

Comment:
Before checking if the inputTokens[0] is not nil, we should check that len(inputTokens) is not 0.